packages/wallet/frontend/src/ui/CopyButton.tsx

```diff
@@ -3,20 +3,22 @@ import { useEffect, useState } from 'react'
 import { Button, ButtonProps } from './Button'
 import { cx } from 'class-variance-authority'
 
-function copyToClipboard(value: string) {
-  navigator.clipboard.writeText(value)
+function copyToClipboard(value: string, copyType?: string) {
+  navigator.clipboard.writeText(copyType === 'base64' ? btoa(value.trim()) : value)
 }
 
 type CopyButtonProps = Omit<ButtonProps, 'intent'> & {
   value: string
   ctaText?: string
   afterCtaText?: string
+  copyType?: string
```

Comment:
We can revert these changes and instead of calling `btoa` in the `copyToClipboard` function, we should the specify the base64 encoded key directly:
```jsx
<CopyButton
  // ...rest
  value={btoa(privateKey)}
/>
```